odoo/addons/base/models/res_users.py

```diff
@@ -907,6 +907,10 @@ def action_show_rules(self):
             'target': 'current',
         }
 
+    def _is_internal(self):
+        self.ensure_one()
+        return not self.sudo().share
```

Comment:
Tests do not crash when I remove this `sudo`. Test coverage should be extended to cover it if it is necessary, or it should be reverted if not.